src/parser/csql_grammar.y

```diff
@@ -3617,7 +3624,7 @@ alter_stmt
 	  INDEX				/* 2 */
 	  identifier			/* 3 */
 	  ON_				/* 4 */
-	  class_name			/* 5 */
+	  user_specified_name		/* 5 */
```

Comment:
class_name and user_specified_name are too different. Is user_specified_name appropriate?

Reply:
How about using user_specified_name in the class_name rule?
In the same way, other object_name is also changed.
- serial_name, trigger_name
```
class_name
        : user_specified_name
        ;
```